server/server.js

```diff
@@ -6,21 +6,46 @@ const path = require('path');
 require('dotenv').config(); //??
 const db = require('../server/db/db-connection.js'); 
 const fetch = require("node-fetch"); 
+const { auth } = require('express-openid-connect');
 const apiKey = `${process.env.API_KEY}`;
 const REACT_BUILD_DIR = path.join(__dirname, '..', 'client', 'build');
 const app = express();
-app.use(express.static(REACT_BUILD_DIR));
 
-const PORT = process.env.PORT || 5001;
+
+
```

Comment:
remove unnecessary extra lines here

Reply:
fixed in [3ad0106](https://github.com/GabyC0/wander/pull/1/commits/3ad0106c4123bcf845bfdd5c81a9be616117d68a)